packages/ui/src/TabItem/TabItem.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+
+import type { SVGComponent } from "@eisbuk/svg";
+
+interface TabItemProps extends React.HTMLAttributes<HTMLButtonElement> {
+  label: string;
+  Icon: SVGComponent;
+  active?: boolean;
+}
+
+const TabItem: React.FC<TabItemProps> = ({
+  label,
+  Icon,
+  onClick,
+  active = false,
+  ...props
+}) => {
+  const onClickHandler = onClick ? onClick : () => {};
+
+  const buttonStateClasses = active
+    ? buttonActiveClasses
+    : buttonDefaultClasses;
+
+  const svgClassNames = svgClasses.join(" ");
+  const buttonClassNames = [...buttonBaseClasses, ...buttonStateClasses].join(
+    " "
+  );
+
+  return (
+    <button className={buttonClassNames} onClick={onClickHandler} {...props}>
```

Comment:
We should be able to pass `className` to the component (like so: `<TabItem {...props} className="absolute left-4" />` or smtn.) and those classes should be passed to a wrapping element (in this case `<button>`). This would mostly be used for positioning, min-w, etc.

Reply:
I see now you collect all the `...props` and spread them into `<button>` element. In case of extending classes (as illustrated above), the custom `className` would override all of the default classes. Therefore classes should be extended (like so `const buttonClassNames = [...aBunchOfDefaultClasses, className].join(" ")`)

Reply:
Ok, noted - I will update!